Backpack/src/main/java/net/skyscanner/backpack/chip/internal/BpkChipStyles.kt

```diff
@@ -38,6 +41,7 @@ internal sealed class BpkChipStyles : BpkChipStyle {
     override var backgroundColor: Int = context.getColor(R.color.bpkBackgroundElevation03)
     override var textColor: Int = context.getColor(R.color.bpkTextPrimary)
     override var disabledBackgroundColor: Int = Color.TRANSPARENT
+    override val style = BpkChip.Style.Default // use default here to avoid breaking changes when `outline` is removed
```

Comment:
Because `BpkOutlineChip` exists as a separate components, rather than a style there's no nice way of introducing a style while outline still exists. This was the least disruptive way. It does mean that if somebody uses an outline chip and changes the style the style will be no longer outline, but given that outline is deprecated and should no longer be used this is acceptable, as using outline AND using a new property are not compatible.